mandatory/4-tax.js

```diff
@@ -17,7 +19,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(productPrice) {
+  var tax = productPrice*1.2;
```

Comment:
This is very good. I would make one change though - here you can actually use the calculateSalesTax function which you wrote above.
This way, the logic to calculate the sales tax only needs to exist in one place. If it needs to change in the future (maybe if the sales tax changes to 30% of the product) - you will only need to change it in one place. This is one of the benefits of using functions.